cli/src/alluxio.org/cli/cmd/exec/test_run.go

```diff
@@ -0,0 +1,88 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package exec
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"alluxio.org/cli/env"
+)
+
+var TestRun = &TestRunCommand{
+	BaseJavaCommand: &env.BaseJavaCommand{
+		CommandName:   "testRun",
```

Comment:
this is the same as the old `bin/alluxio runTests`?

my preference is to call it `basicIOTests`. don't need `run` because the parent command is `exec` which implies something is being run. prefer `basicIO` over `sanity` because it is more objective. people may have different opinions on what is considered to be "sanity" level.

Reply:
yes this is the old `alluxio runTests`. i think `basicIOTests` is very clear. and maybe we can name other commands to `xxxTests` for consistency.